src/pybind/mgr/dashboard/module.py

```diff
@@ -406,6 +411,40 @@ def set_mgr_created_self_signed_cert(self):
             return result
         return 0, 'Self-signed certificate created', ''
 
+    @CLICommand("dashboard get issue")
+    def get_issues_cli(self, issue_number: str):
+        if issue_number == 'None':
+            return -errno.EINVAL, '', 'Invalid arguments. Issue number is missing'
+        cephTrackerClient = CephTrackerClient()
+        try:
+            response = cephTrackerClient.get_issues(issue_number)
+        except RequestException as error:
+            if error.status_code == 404:
+                return errno.EINVAL, '', 'Issue not found'
+            else:
+                return errno.EINVAL, '', 'Unknown error occurred: ' + str(error)
+        return 0, str(response), ''
+
+    @CLICommand("dashboard create issue")
+    def report_issues_cli(self, project: str, tracker: str, subject: str, description: str):
```

Comment:
You should use an enum here. The CLICommand backend will automatically validate the user input for you in this case. The command `help` line will be auto-generated based on this info, too.

```suggestion
class TrackerType(Enum):
    BUG = 'bug'
    FEATURE = 'feature'

    def report_issues_cli(self, project: str, tracker: TrackerType, subject: str, description: str):
```

References:
* https://github.com/ceph/ceph/blob/master/src/pybind/mgr/dashboard/plugins/motd.py#L15
* https://github.com/ceph/ceph/blob/master/src/pybind/mgr/dashboard/plugins/motd.py#L58

Reply:
As per IRC conversation,  this can be done in the [follow-up PR](https://github.com/ceph/ceph/pull/42603).